sprint8/src/pages/free-board/create-post.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import style from '@/src/styles/create-post.module.css';
+import Header from '@/src/components/Header.js';
+import Button from '@/src/components/Button.js';
+import CreateEditPost from '@/src/components/CreateEditPost.js';
+import axios from '@/src/lib/axios.js';
+
+export default function CreatePost() {
+  const router = useRouter();
+  const [isButtonActive, setIsButtonActive] = useState(false);
+  const [title, setTitle] = useState('');
+  const [content, setContent] = useState('');
+
+  async function postArticle() {
+    const data = { title: title, content: content };
+
+    if (title && content) {
+      try {
+        const res = await axios.post('/articles', data);
+        const articleId = res.data.id;
+        router.push(`/free-board/${articleId}`);
+      } catch (error) {
+        console.error('Error posting article:', error);
+      }
+    }
+  }
+
+  useEffect(() => {
+    if (title && content) {
+      setIsButtonActive(true);
+    } else {
+      setIsButtonActive(false);
+    }
```

Comment:
@annylee0723 이건어떨까요?
```suggestion
    const isButtonActive = title && content
      setIsButtonActive(isButtonActive);
    }
```

Reply:
영감 받고 truthy falsy 적용 후 수정했습니다 
commit: [d71273e](https://github.com/codeit-sprint-fullstack/2-Sprint-Mission-FE/pull/148/commits/d71273e108012bf0f2d2f098cbba27a52f1177f6)

Reply:
@annylee0723 아.. 요거는 이렇게할필요가 없어요 코멘트 다시 남길게요